data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -508,6 +508,16 @@
       [ "lye_powder", 100 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "mutagen_jabberblood", "//": "jabberwocks have a few of these pumping in them, and they can be crushed down for a lot of mutagen",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_MUTAGEN",
+  "skill_used": "survival",
+  "difficulty": 2,
```

Comment:
Considering the... advanced form of this nether creature, or "aberration" as it's defined in the monster.json, I feel like this might require a bit survival knowledge? Then again, is the survivor basically just liquefying the entire heart or is there some pieces that are being removed, etc?

Reply:
It's closer to simply cutting open the heart, extracting the congealed blood, and discarding what remains of it. I might add a byproduct to show that, come to think of it. A tool such as cutting would certainly make sense, though, as the meatiness of the heart would be hard to get through with just bare hands.

It's not *difficult* to extract the blood, which is why I set it to survival 2. I could bump it up to 3 or so, if wanted.